imagej/imagej.py

```diff
@@ -630,6 +630,18 @@ def _format_value(self, value):
             final_value = '[' + temp_value + ']'
             return final_value
 
+        def get_window_manager(self):
+            """
+            Get the ImageJ1 window manager if legacy mode is enabled.  It may not work properly if in headless mode.
+            :return: WindowManager
+            """
+            if not ij.legacy_enabled:
+                raise ImportError("Your ImageJ installation does not support IJ1.  This function does not work.")
+            elif ij.ui().isheadless():
+                logging.warning("Headless mode not enabled.  The WindowManager may not function as expected.")
```

Comment:
Shouldn't this read "Headless mode enabled."?

Reply:
You are correct.